templates/index.html

```diff
@@ -88,7 +89,11 @@ <h1>סמנטעל</h1>
 <footer>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
     <script src="https://cdn.statically.io/gh/zenorocha/clipboard.js/v2.0.10/dist/clipboard.min.js"></script>
-    <script src="js/semantle.js?version={{js_version}}"></script>
+    <script src="static/semantle.js?version={{js_version}}"></script>
+
+    <script src="//static/semantle.js?version={{js_version}}"></script>
```

Comment:
this is wrong, leave only the first `script` (here and elsewhere)

Reply:
https://github.com/ishefi/semantle-he/commit/c87f89be6451ce4e7e5f810b019adb7b38badd2f